src/components/AboutComponent/AboutComponent.tsx

```diff
@@ -1,24 +1,23 @@
 import React from 'react'
 import './AboutComponent.scss'
 import { Helmet } from 'react-helmet'
+import { useTranslation } from "react-i18next";
 
 const About: React.FC = () => {
+
+    const {t} = useTranslation();
+
     return (
         <div className="about">
             <Helmet>
-                <title>CConverter - O projekcie</title>
+                <title>CConverter - About Project</title>
```

Comment:
This change will be visible in browser tab. Aswell it can be used by google search engine and affect the results of page description. 
That's why I think we should also add translations to the Helmet meta tags. This concerns all pages.

Reply:
Done

Reply:
Done